src/pages/Users/index.jsx

```diff
@@ -106,11 +107,17 @@ const Users = () => {
       )
     },
     {
-      Header: 'Created',
+      Header: useFormatMessage('Users.created'),
       accessor: 'created',
       Cell: ({ row }) => (
         <small className="has-text-grey is-abbr-like">
-          {row.original.createdAt}
+          {useFormatDate(
+            row.original.createdAt,
```

Comment:
The same here, it's really necessary to change the default date format options?.